quickwit/quickwit-proto/src/quickwit.rs

```diff
@@ -199,10 +199,10 @@ pub struct LeafSearchResponse {
     /// num_attempted_splits = num_successful_splits + num_failed_splits.
     #[prost(uint64, tag = "4")]
     pub num_attempted_splits: u64,
-    /// json serialized intermediate aggregation_result.
-    #[prost(string, optional, tag = "5")]
+    /// postcard serialized intermediate aggregation_result.
+    #[prost(bytes = "vec", optional, tag = "5")]
     pub intermediate_aggregation_result: ::core::option::Option<
-        ::prost::alloc::string::String,
+        ::prost::alloc::vec::Vec<u8>,
```

Comment:
We should `Bytes`. It's better because it's cheaply cloneable. It makes a difference when coupled with some retry logic that potentially copies the same request multiple times.